setup.py

```diff
@@ -2,17 +2,18 @@
 
 setup(
     name='drf-oidc-auth',
-    version='0.10.0',
+    version='0.11.0',
```

Comment:
Wondering if this might need to be upped to `1.0.0` because of the cryptography/setting change.

Reply:
We could update to `1.0.0` :).

Reply:
I think that is required, not because of the updated cryptography requirement though (I think it's fine if we just put it in the `install_requires` in `setup.py`). But since the `OIDC_AUDIENCES` is removed in favour of `OIDC_CLAIMS_OPTIONS`. I think that is not a backwards compatible change. However, if we make it backwards compatible as you mentioned in the comment below we can leave it like this. I guess it depends on what we want to do. It would perhaps be nice to have a backwards compatible version, but not sure if it's really worth doing in this case. Probably fine to leave it and just upgrade to `1.0.0`

Reply:
I think something changed in regards to the `OIDC_CLAIMS` behaviour. Previously if an audience was not in the `OIDC_AUDIENCES` an error would be returned (see https://github.com/ByteInternet/drf-oidc-auth/blob/de85585d3a4f742aedf77ef8fd8d187cf893bd51/oidc_auth/authentication.py#L153). Right now I think any audience is able to make a request independent of which audiences are specified in the settings. Because I have not specified any audiences in the settings but am still able to make a request and receive a response independent on which audience does the request.

Reply:
its now `1.0.0`